packages/serverless/src/index.ts

```diff
@@ -1,10 +1,21 @@
-// https://medium.com/unsplash/named-namespace-imports-7345212bbffb
-import * as AWSLambda from './awslambda';
-import * as GCPFunction from './gcpfunction';
+export { awsServicesIntegration } from './awsservices';
+import { init as awsLambdaInit, tryPatchHandler, wrapHandler } from './awslambda';
+import { init as gcpFunctionInit } from './gcpfunction';
+
+const AWSLambda = {
+  init: awsLambdaInit,
+  wrapHandler,
+  tryPatchHandler,
+};
+
+const GCPFunction = {
+  init: gcpFunctionInit,
+};
```

Comment:
This may inhibit tree shaking.

Reply:
Yeah this is fine, I'm trying to map to old behaviour as much as possible.